Controllers/Web/v1/UsersController.cs

```diff
@@ -66,6 +68,26 @@ public async Task<IActionResult> GetProfileByIdAsync([FromRoute]long id)
             return NotFound(CreateErrorResponse(StatusCodes.Status404NotFound.ToString(), "NotFound"));
 
         }
+
+        // I need your opinion on updating about for user :)
+        // Add it here? Or add it to every controller?
+        [HttpPatch("about")]
+        public async Task<IActionResult> UpdateAboutAsync([FromBody] UpdateUsersAboutDTO updateUsersAboutDTO)
+        {
+            if (!ModelState.IsValid)
+                return CustomBadRequest();
+
+            var identity = HttpContext.User.Identity as ClaimsIdentity;
+            long authenticatedUserId = authService.GetUserId(identity);
```

Comment:
getting the userId from authService.GetUserId, is not useful here, instead you can use `userManager.GetUserAsync(HttpContext.User)`.

Reply:
we don't have UserService :)

Reply:
Done